plugins/hydra_ray_launcher/hydra_plugins/hydra_ray_launcher/_launcher_util.py

```diff
@@ -121,15 +116,16 @@ def ray_down(yaml_path: str) -> None:
     _run_command(["ray", "down", "-y", yaml_path])
 
 
-def ray_up(yaml_path: str) -> None:
-    _run_command(["ray", "up", "-y", yaml_path])
+def ray_up(yaml_path: str, no_config_cache: bool) -> None:
+    args = ["ray", "up", "-y", yaml_path]
+    if no_config_cache:
+        args.append("--no-config-cache")
```

Comment:
do we always need `--no-config-cache`? if not we should expose this in the launcher config?

Reply:
Also, what is --no-config-cache for?

Reply:
I added the option because the cached AWS config from running the integration tests was interfering with running jobs with my normal AWS profile. We probably could just always pass the flag, I just assumed we'd want to make it an option instead of hardcoding it

Reply:
Could you provide a repro or stacktrace for this? 
Ray's documentation doesn't have much info on this option, I'd like to understand a bit more here :)

Reply:
also if we are exposing the `--no-config-cache` option through the config, we need to think about should we only apply it in `ray up`

since this option can be used in combo with other major command like `exec`, `submit` etc
https://docs.ray.io/en/releases-1.1.0/package-ref.html#cmdoption-ray-up-no-config-cache

Reply:
>I added the option because the cached AWS config from running the integration tests was interfering with running jobs with my normal AWS profile. We probably could just always pass the flag, I just assumed we'd want to make it an option instead of hardcoding it


I'm not sure I understand the issue here. Could you be more specific?